packages/fixtures/src/news-and-events.ts

```diff
@@ -1,10 +1,11 @@
-import { NewsOrEventResponse } from '@asap-hub/model';
+import { NewsAndEventsType, NewsOrEventResponse } from '@asap-hub/model';
 
 export const createNewsAndEventsResponse = (
   key: string,
+  responseType: NewsAndEventsType = 'News',
 ): NewsOrEventResponse => ({
   id: `uuid-${key}`,
-  type: 'News',
+  type: responseType,
   title: `News ${key} title`,
```

Comment:
I think we should put `responseType` in here too as events will show up as news in storybook or tests